doc/Quick_start/deployment.md

```diff
@@ -1,6 +1,7 @@
-# Deploying Contrail with Windows compute nodes
+# Heterogenous Contrail Deployment: Linux + Windows
```

Comment:
It suggests setup with windows and linux compute nodes (at least to me). What's wrong with "deploying contrail controller with windows compute nodes"?